editors/code/tsconfig.json

```diff
@@ -1,7 +1,7 @@
 {
     "compilerOptions": {
         "module": "commonjs",
-        "target": "es2018",
+        "target": "es2019",
```

Comment:
Where do I look which es version is supported on node which is shipped with VS Code? Like, how can I check what's the highest target we can use here?

Reply:
It doesn't seem clear-cut, but https://node.green/#ES2019.

Reply:
@matklad for reference, this is the table I was looking at: https://kangax.github.io/compat-table/es2016plus/#chrome78. You will have to "show obsolete platforms" checkbox in the top left.